packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,524 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+This package is **deprecated** and will no longer be maintained. All `Porsche Design System` utilities are now provided
+at`@porsche-design-system/components-{js|angular|react|vue}/styles` in from of styles and Design Tokens. To make the
+migration easier, we offer an overview of the old deprecated values in reference to the new design tokens. Further
+documentation about the new Design Tokens can be found here
```

Comment:
A bit mixed with `design tokens` and `Design Tokens`. 
I personally would go with the lower case version since it isn't a component, package name or anything.